### PR TITLE
Vwrs configure edgeworker

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds instructions for configuring a VWRs EdgeWorker.

### CODE REVIEW

File: docs/photoniq/vwrs/configure-vwrs-edgeworker.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+   https://api-vwr-service.gdn-akamai.com/api/vwr/v1/origins/example3.com \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'authorization: test_key.12345678abcdefghijklABCDEFGHIJKL' \
+  -d '{
+  "priority": [1, 20, 255]
+}'
+```
+
+You have now successfully configured the waiting room to acknowledge priorities. To prioritize the traffic, you must pass the priority to the `handleVwrsRequest()` and `handleVwrsResponse()` functions. Here's a template demonstrating how to include a priority with the request in the `handleVwrsRequest()` call:
```
Comment (joubert-macrometa) [inline docs/photoniq/vwrs/configure-vwrs-edgeworker.md]: "...to acknowledge priorities."  @rahul-naik1, what does "acknowledge priorities." mean?
Do we mean 
"You have now successfully configured the waiting room to support priorities."

**Review by joubert-macrometa — APPROVED** [review]
I am marking this as approved, but @rahul-naik1 needs to verify the one question I have.

File: docs/photoniq/vwrs/configure-vwrs-edgeworker.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+title: Configure a VWRs EdgeWorker
+---
+
+Import the `VirtualWaitingRoom` class from the virtualWaitingRoom.js file to configure a waiting room.
+
+## Mandatory Configuration Options
+
+The following configuration options must be set:
+
+- **apiKey**: This API Key is generated using the VWRs authorization key API. Please refer to the documentation for `POST /api_vwr/v1/apikey` on how to create this key.
```
Comment (rahul-naik1) [inline docs/photoniq/vwrs/configure-vwrs-edgeworker.md]: POST /api/vwr/v1/apikey

File: docs/photoniq/vwrs/configure-vwrs-edgeworker.md (hunk 1 above)
Comment (rahul-naik1) [reply docs/photoniq/vwrs/configure-vwrs-edgeworker.md]: I agree with @joubert-macrometa Let's update it.

**Review by rahul-naik1 — APPROVED** [review]
Rest of the changes looks fine

**Pull request merged**